README.md

```diff
@@ -1 +1,162 @@
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#show-your-support)
+- [🙏 Acknowledgements](#acknowledgments)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# About Project
+
+BballGeeks is a comprehensive mobile application designed for basketball enthusiasts and NBA fans. This app provides users with access to a wealth of NBA statistics spanning over 10 seasons. Whether you're a casual fan or a dedicated basketball analyst, BballGeeks offers a wide range of features and data to satisfy your basketball cravings.
+
+## Built With
+
+### Tech Stack 
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="">REACT</a></li>
+    <li><a href="">REDUX</a></li>
+    <li><a href="">CSS</a></li>
+  </ul>
+</details>
+
+### Key Features
+
+
+- **[REACT]**
+- **[REDUX]**
+- **[API]**
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need: 
+- GitHub;
+- Code Editor;
+- Basic Knowledge of HTML and JavaScript.
+
+
+### Setup
+
+Clone this repository to your desired folder:
+```sh
+  cd my-folder
+  git clone https://github.com/RicardoEgas/BballGeeks.git
+```
+
+
+### Install
+
+Install this project with:
+- npm install
+
+### Usage
+
+To run the project, execute the following command:
+
+- npm start
+
+### Run tests
+
+To run tests, run the following command:
+
+- npx eslint "**/*.{js,jsx}"
+- npx stylelint "**/*.{css,scss}"
+
+### Live Demo
+
+[Demo](https://ricardoegas.github.io/BballGeeks)
+
+### Deployment
+
+You can deploy this project using:
+
+GitHub Pages
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Authors 
+
+
+👤 **Ricardo Egas**
+
+- GitHub: [@githubhandle](https://github.com/RicardoEgas)
+- Twitter: [@twitterhandle](https://twitter.com/RjEgas)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ricardo-egas-952125136)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+##  Future Features 
+
+
+- [ ] **[Styling]**
+- [ ] **[Improved UI]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+##  Contributing <a name="contributing"></a>
+
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+##  Show your support 
+
+
+If you like this project give me a star⭐️!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+##  Acknowledgments
+
+I would like to thank Microverse!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] The readme file is also expected to contain an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-\(Free-PSDs\)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).